Cargo.toml

```diff
@@ -21,5 +21,8 @@ serde_urlencoded = "*"
 http = "0"
 bytes = "*"
 angry-purple-tiger = "0"
-helium-crypto = { git = "https://github.com/helium/helium-crypto-rs", tag = "v0.3.4", features = ["ecc608"]}
+helium-crypto = { git = "https://github.com/OpenEPC/helium-crypto-rs", branch = "tpm", optional = true }
```

Comment:
This shouldn't be pointing at another repo.. is this just while the tmp code is merged up into helium-crypto-rs?

Reply:
yes. I'll update this and gateway-rs pull requests once helium-crypto PR is approved.